packages/tatum-ledger/src/account.ts

```diff
@@ -53,7 +54,7 @@ export const generateAccount = async (
   let w
   if (generateNewWallet) {
     // @ts-ignore
-    w = await generateNewWalletFn(testnet)
+    w = await generateNewWalletFn({ testnet })
```

Comment:
what about mnemonic?

Reply:
This did not seem to me as part of the original scope (check if it will work), but I can extend generateAccount with mnemonic input as well.

Reply:
I checked that the original version has no `mnemonic`, it takes only `testnet`.